contracts/smart-wallet/non-upgradeable/Account.sol

```diff
@@ -56,6 +60,7 @@ contract Account is
     receive() external payable virtual {}
 
     constructor(IEntryPoint _entrypoint) {
+        factory = msg.sender;
```

Comment:
Is this safe? What if i deploy manually?

Reply:
I now accept the factory address as an argument, and you can safely pass `address(0)` if you're deploying an account directly.